WikiDrips/WikiDocViewController.swift

```diff
@@ -49,3 +58,14 @@ class WikiDocViewController: UIViewController, WKUIDelegate {
 
 
 }
+
+extension WikiDocViewController: WKNavigationDelegate {
```

Comment:
I see a slight UI bug where the indicator is still visible when content starts to load. Maybe you also want to stop it in: `func webView(_ webView: WKWebView, didCommit navigation: WKNavigation!)`